geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala

```diff
@@ -104,8 +98,11 @@ class ConverterIngest(dsParams: Map[String, String],
               toWrite.getUserData.put(Hints.USE_PROVIDED_FID, java.lang.Boolean.TRUE)
               try {
                 fw.write()
+                written.incrementAndGet()
```

Comment:
I guess technically we have to wait unless the flush happens on the BW to claim its truly done...but I feel like we don't really want to deal with that unless we have to....you have to keep a list of features that are added to the BW and then clear the list when you call flush(). Similar to we we've done in storm to ack tuples.